plugins/cql-language/src/test/java/org/polypheny/db/cql/utils/CqlInterfaceTest.java

```diff
@@ -16,58 +16,73 @@
 
 package org.polypheny.db.cql.utils;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
 import com.google.gson.JsonObject;
 import java.sql.Date;
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 import kong.unirest.HttpRequestWithBody;
 import kong.unirest.HttpResponse;
 import kong.unirest.JsonNode;
 import kong.unirest.Unirest;
 import kong.unirest.json.JSONArray;
 import kong.unirest.json.JSONObject;
 import lombok.extern.slf4j.Slf4j;
-import org.junit.Assert;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.polypheny.db.TestHelper;
 import org.polypheny.db.cql.utils.helper.CqlTestHelper;
 
 
 @Slf4j
 public class CqlInterfaceTest extends CqlTestHelper {
 
+    private static TestHelper helper;
+
+
+    @BeforeAll
+    public static void setUp() {
+        helper = TestHelper.getInstance();
+
+    }
+
+
     @Test
-    @Ignore
+    @Disabled
     // todo rewrite in httpinterface test
     public void testRestCqlEmptyQueryReturnsException() {
         JsonNode expectedJsonNode = new JsonNode( "{\"Exception\":\"CQL query is an empty string!\"}" );
-        cqlInterfaceTestHelper( "", expectedJsonNode );
+        cqlInterfaceTestHelper( "", expectedJsonNode, true );
     }
```

Comment:
todo

Reply:
Refactored test and added missing error handling during query processing for empty queries.